runtime/js-compute-runtime/fastly.wit

```diff
@@ -486,7 +486,7 @@ default world fastly-world {
     purge-surrogate-key: func(surrogate-key: string, soft-purge: bool) -> result<purge-result, error>
   }
 
-  export js-compute-runtime: interface {
+  export compute-at-edge: interface {
```

Comment:
This change was necessary as we need to name the exported interface in the host side as well. Happy to change to a different name, but this seemed generic enough for multiple sdks to target.